runtime/lib/backend/openqasm/braket.local.qubit.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+version = "v0.31.0"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+[[operations.gates]]
+full = [
```

Comment:
Not all of them are supported! Check the complete list https://github.com/PennyLaneAI/catalyst/blob/c972eb779cdd310306f2b9a387ac2a05061b1732/runtime/lib/backend/openqasm/OpenQasmBuilder.hpp#L71

We can revisit the list of supported gates for these devices too.